user-journey-tests/helpers/testHelpers/checkAnswers.js

```diff
@@ -137,17 +137,19 @@ export const validateReceiveMethod = async (changeLink, valueElement) => {
   await expect(receiveMethodPage.emailRadio).toBeSelected()
   await receiveMethodPage.selectEmailAndContinue()
 
-  await validateElementVisibleAndText(valueElement, 'email')
+  await validateElementVisibleAndText(valueElement, 'Email')
```

Comment:
@joseluisgraa  - huh - shouldn't this mean that publish should have been failing since we changed 'email' to 'Email' (which was quite a while ago)?

This looks like the right fix, but it's a mystery to me why we weren't having only total failures without this fix in.

Reply:
I may be answering my own question - we have, and I've not been paying enough attention: https://github.com/DEFRA/apha-apps-perms-move-animal-ui/actions/workflows/publish.yml

Reply:
Yes it is a mistery, there was another place where we seemed to be using a test-id that I couldn't see in the code...